awx/ui_next/src/components/LaunchPrompt/useSteps.js

```diff
@@ -13,14 +13,20 @@ export default function useSteps(config, resource, i18n) {
     useOtherPromptsStep(config, resource, visited, i18n),
     useSurveyStep(config, resource, visited, i18n),
   ];
+  const formErrors = steps
+    .filter(s => s.formError != null)
+    .filter(s => s !== undefined)
+    .filter(s => Object.keys(s.formError).length > 0);
```

Comment:
it's better to just chain these with &&, that way it doesn't have to iterate over the steps 3 times (not that it really matters in this case, because the number of steps is not large, but it's a better convention)